samples/Config.CustomConfigurationProviders.Web/Extensions/ConfigurationProviderKeyExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.Configuration.Extensions
+{
+    public static class ConfigurationProviderKeyExtensions
+    {
+        /// <summary>
+        /// Gets the full key to search for in an IConfigurationRoot
+        /// </summary>
+        /// <example>
+        /// Given the configuration file:
+        /// <configuration>
+        ///     <nodea>
```

Comment:
Should check if this doc comment actually works. I'm guessing you need to XML-encode it (basically same as HTML-encode), otherwise none of these tags will show up.